### ISSUE TITLE
Version 4.21 has bug to translate

### ISSUE THREAD

**Issue by medeiroz** [knuckleswtf/scribe#676]
### Scribe version

4.21

### PHP version

8.1

### Framework

Laravel

### Framework version

10.12.0

### Scribe config

```ruby
title => "MS.Indicacao Documenta\u00e7\u00e3o"
base_url => "http://localhost:8888"
type => "laravel"
auth.enabled => true
auth.default => true
```

### What happened?

I ran `php artisan scribe:genrate` the error `"Translation not found for $key. You can add a translation for this in your `lang/scribe.php`, but this is likely a problem with the package. Please open an issue."` is shown

I rollback to 4.20 and it worked as expected

### Docs

- [X] I've checked [the docs](https://scribe.knuckles.wtf/laravel), the [troubleshooting guide](https://scribe.knuckles.wtf/laravel/troubleshooting), and [existing issues](https://github.com/knuckleswtf/scribe/issues?q=is%3Aissue+), but I didn't find a solution

**Comment by brecht-vermeersch** [comment]
We have the same issue

**Comment by shalvah** [comment]
Do you use custom views or something? Did you publish the previous translation files? Also see the docs on localisation. https://scribe.knuckles.wtf/laravel/advanced/localization

**Comment by shalvah** [comment]
Also, can you share the exact exception that was thrown? This isn't the actual message.

**Comment by ccec1t91077** [comment]
I did some trace, it might be a environment issue.

The `$this->app->runningInConsole()` [here](https://github.com/knuckleswtf/scribe/blob/master/src/ScribeServiceProvider.php#L59) will return false in some case (e.g. PHP-CGI), and it will trying to use default loader instead of the `CustomTranslationsLoader`, then the error will thrown by [this line](https://github.com/knuckleswtf/scribe/blob/master/src/ScribeServiceProvider.php#L59).

**Comment by shalvah** [comment]
Hmm, interesting. Are you using Laravel Octane? How are you running the generate command?

**Comment by shalvah** [comment]
Okay, I think I've found a fix. Can you guys try with the `master` branch now? `composer require --dev knuckleswtf/scribe:dev-master` should be enough.

**Comment by bmbBAMBUS** [comment]
I had the same issue, I can cofirm that `dev-master` works.

**Comment by shalvah** [comment]
Fixed in 4.21.1

**Comment by JonPurvis** [comment]
Thanks @shalvah 👍
